bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -321,6 +321,9 @@ public void createTable(HTableDescriptor desc, byte[][] splitKeys) throws IOExce
   }
 
   /**
+   * To create Table.
```

Comment:
How about "Create a Table"

Reply:
Thanks for quick feedback, Updated with the same.